scripts/globals/abilities/martyr.lua

```diff
@@ -43,7 +43,7 @@ function onUseAbility(player,target,ability)
     -- Log HP Headed for Debug
     -- printf("Martyr Healed HP: %d", healHP)
 
-    player:delHP(damageHP)
+    player:takeDamage(damageHP)
```

Comment:
Stoneskin here, also.
`Stoneskin can mitigate the amount of HP sacrificed. However, the ability does not lose potency this way.`